Microsoft.AVS.NFS/Microsoft.AVS.NFS.psm1

```diff
@@ -0,0 +1,156 @@
+using module Microsoft.AVS.Management
+<#
+    .SYNOPSIS
+     This function mounts the NFS datastore on all hosts in the cluster.
+
+    .PARAMETER ClusterName
+     Cluster name
+
+    .PARAMETER DatastoreName
+     Name of the NFS Datastore to be mounted on hosts in the cluster.
+    
+    .PARAMETER NfsSharePath
+     Path to NFS share.
+    
+    .PARAMETER NfsHost
+     IP of NFS server.
+
+    .EXAMPLE
+     New-NFSDatastore -ClusterName Cluster1 -DatastoreName testfileshare -NfsHost 172.24.1.8 -NfsSharePath /ankstorageacct/testfileshare/
+
+    .INPUTS
+     vCenter cluster name, Name of the NFS datastore, Path to the NFS share, IP of the NFS server.
+
+    .OUTPUTS
+     None.
+#>
+function New-NFSDatastore {
+    [CmdletBinding()]
+    [AVSAttribute(10, UpdatesSDDC = $false)]

```

Comment:
Would these cmdlets need to be shown in Azure portal? If not, I would recommend adding the AVSAttribute `AutomationOnly = $true`